src/opentime/rationalTime.cpp

```diff
@@ -10,15 +10,17 @@ namespace opentime { namespace OPENTIME_VERSION {
 
 RationalTime RationalTime::_invalid_time{ 0, RationalTime::_invalid_rate };
 
-static constexpr std::array<double, 4> dropframe_timecode_rates{ {
+static constexpr std::array<double, 6> dropframe_timecode_rates{ {
     // 23.976,
     // 23.98,
     // 23.97,
     // 24000.0/1001.0,
     29.97,
     30000.0 / 1001.0,
+    48000.0 / 1001.0,
```

Comment:
Sadly, field recorders also offer 48000 * 1001 / 1000 as a rate, so it should be added as well.

Reply:
That is the record speed, but what timecode do they write?

Reply:
Right, right. These values are meant for rates with associated SMPTE time codes, which does not include audio data. For some reason I was thinking that all the NTSC related rates were part of this stew, but they aren't.